Update Emerson/Vertiv Controllers

craig-nokia opened the pull request:
Vertiv/Emerson controller updates to handle embedded linux and updated the sensors.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

craig-nokia commented on resources/definitions/os_discovery/ees.yaml:
```diff
@@ -1,219 +1,240 @@
+          descr: 'SM IO Modules'
+          skip_values: 2147483647
+
+        - oid: numberOfInstalledRectifiers
```
I removed the count: section as it does not make sense for alarming.

murrant commented:
Curious, why did you delete the snmprec file? Can you generate the json test files?

craig-nokia commented:
> Curious, why did you delete the snmprec file? Can you generate the json test files?

I updated the resources/definitions/os_discovery/ees.yaml and resources/definitions/os_discovery/ees.yaml, I found that we did not need the old and new controller tests.    I deleted the old tests and replace with tests/snmpsim/ees_controller.snmprec and tests/data/ees_controller.json.

laf reviewed (approved):
I've updated the test data, thanks for contributing @craig-nokia
